src/js/components/Skeleton/stories/Group.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+
+import { Box, Button, Card, Grommet, Image, Text, ThemeContext} from 'grommet';
+import { FormAdd } from 'grommet-icons';
+
+const Item = ({ ...rest }) => (
+  <Card
+    width="531px"
```

Comment:
Same as other comment, can we avoid `px` values?

Reply:
I made the GridContainer example do tshirt sizes for the Card but  for the Group and Individual examples it wasn't looking good or representing what we mocked. It's that awkward size between medium and large.